### DISCUSSION TITLE
Standalone header evaluation library based on mdn-http-observatory

### DISCUSSION THREAD

**Discussion by lfarrel6** [mdn/mdn-community#759] [Platform]
The tooling for evaluating the HTTP Headers returned from a service using the HTTP Observatory project is really useful, and, from what I can see, is one of the few open source http header-scoring projects. The project itself is structured as a complete service to be run as a REST API. I was wondering if there was an appetite for refactoring the project into a header scoring library, with a separate project for the service? I think the overall focus of improving http header hygiene is extremely useful, and could benefit from the flexibility of a dedicated lib.

**Comment by pransh15** [comment]
Hi @lfarrel6 👋 

Thanks for your kind words! Your suggestion is interesting. It could make it more useful for different types of implementations. I'd be curious to hear if others in the community are interested in contributing to such a refactor, or if there's appetite for maintaining it as a separate lib. I'll relay the idea to the engineering team, and something we will discuss further.

**Reply by lfarrel6** [reply]
Thanks, @pransh15 🙌
